aioftp/errors.py

```diff
@@ -2,13 +2,21 @@
 
 
 __all__ = (
+    "AIOFTPError",
     "StatusCodeError",
     "PathIsNotAbsolute",
     "PathIOError",
+    "NoAvailablePort",
 )
 
 
-class StatusCodeError(Exception):
+class AIOFTPError(Exception):
```

Comment:
This should be `AIOFTPException`.

Reply:
Why? `IndexError`, `KeyError` ...

Reply:
`Exception`, `BaseException` :)
Error should be concrete, exception can be abstract.

Reply:
Agree, renamed.